packages/wasm/src/transaction.ts

```diff
@@ -6,7 +6,7 @@ import {
   TransactionView,
 } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1/transaction_pb';
 
-export const transactionInfo = async (
+export const generateTransactionInfo = async (
```

Comment:
I renamed it because it caused conflicts with RPC `transactionInfo`

Reply:
We will need to split the `generateTransactionInfo()` function into two separate ones in the future 
1. Create `TransactionPersective` from `Transaction`
2. Generate `TransactionView` from `TransactionPersective` 
This will be needed to add prices to the `TransactionPersective` at an intermediate stage. see https://github.com/penumbra-zone/web/issues/557

I didn't do it in this PR because ideally we need to implement tests for wasm first https://github.com/penumbra-zone/web/issues/693